sumologic/sumologic_client.go

```diff
@@ -447,6 +447,7 @@ type Connection struct {
 	DefaultPayload    string    `json:"defaultPayload"`
 	WebhookType       string    `json:"webhookType"`
 	ConnectionSubtype string    `json:"connectionSubtype,omitempty"`
+	ResolutionPayload *string   `json:"resolutionPayload"`
```

Comment:
Made resolution payload a string pointer instead of string because, i didn't see another option to send null for resolution payload unless specifying string pointer in go.